go/vt/vttablet/tabletmanager/vreplication/controller.go

```diff
@@ -99,21 +98,20 @@ func newController(ctx context.Context, params map[string]string, dbClientFactor
 
 	state := params["state"]
 	blpStats.State.Store(state)
```

Comment:
A previous change had resulted in this function returning early for Stopped workflows before the `source` proto was initialized resulting in empty source keyspace/shard in some vreplication debug var stats.